<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>259</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="3">
    <layout class="QHBoxLayout" name="lo_format">
     <item>
      <widget class="QLabel" name="lb_format">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Select format : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_format">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>mp4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mp3</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1" rowspan="4">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <layout class="QHBoxLayout" name="lo_path">
     <item>
      <widget class="QLabel" name="lb_path">
       <property name="text">
        <string>output path : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_path"/>
     </item>
     <item>
      <widget class="QToolButton" name="btn_pathfinder">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="4">
    <layout class="QHBoxLayout" name="lo_link">
     <item>
      <widget class="QLabel" name="lb_link">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Youtube Link : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_link"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QDialogButtonBox" name="btn_exit">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" rowspan="4">
    <layout class="QVBoxLayout" name="lo_checkbox">
     <item>
      <widget class="QCheckBox" name="cb_custom_file_name">
       <property name="text">
        <string>custom file name</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_thumbnail">
       <property name="text">
        <string>with thumbnail</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_subtitle">
       <property name="text">
        <string>with subtitle</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lb_option">
     <property name="text">
      <string>Option</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <layout class="QHBoxLayout" name="lo_filename">
     <item>
      <widget class="QLabel" name="lb_filename">
       <property name="text">
        <string>filename:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_filename">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="3">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
